### ISSUE TITLE
[Android 11] [Crash] When user leaves captive_portal_https_url field blank and taps on save button

### ISSUE THREAD

**Issue by shailendramaurya** [MuntashirAkon/CaptivePortalController#1]
The app get crashed, when the user leaves 
*Captive Portal HTTPS URL* field blank and then taps on Save button.

Device: Xiaomi POCO M3, Android 11

**Comment by shuvashish76** [comment]
Crash if
1. If all Custom Servers fields left empty > Save
2. If you fill any one of the Custom Servers field but others remains empty > Save (no crash if you fill 2 or more field)
3. Fill all field > save > force stop > reopen > click save with empty field (similar to case1 )

**Crash logs**
```
FATAL EXCEPTION: main
Process: io.github.muntashirakon.captiveportalcontroller, PID: 20332
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.startsWith(java.lang.String)' on a null object reference
	at io.github.muntashirakon.captiveportalcontroller.ConnectivityManager.setCaptivePortalServers(ConnectivityManager.java:200)
	at io.github.muntashirakon.captiveportalcontroller.MainActivity.lambda$onCreate$0$io-github-muntashirakon-captiveportalcontroller-MainActivity(MainActivity.java:92)
	at io.github.muntashirakon.captiveportalcontroller.MainActivity$$ExternalSyntheticLambda0.onClick(Unknown Source:4)
	at android.view.View.performClick(View.java:6294)
	at android.view.View$PerformClick.run(View.java:24774)
	at android.os.Handler.handleCallback(Handler.java:790)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6518)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Device details**
App version: 1.1.0 (3)
Android: 8.1.0
Model: Lenovo TB-8704X

**Comment by MuntashirAkon** [comment]
Fixed in 7b45f7047fb3bc7443d7844154868595d4dd7138
